commons/c2cgeoportal_commons/alembic/env.py

```diff
@@ -50,6 +50,25 @@ def get_config():
         'version_locations': context.config.get_main_option('version_locations'),
         'version_table_schema': config[schema_config_name],
     })
+
+    # Autogenerate config
+    _schema = False
+    if alembic_name == 'main':
+        from c2cgeoportal_commons.models.main import Base, _schema
+    elif alembic_name == 'static':
+        from c2cgeoportal_commons.models.static import Base, _schema
+    def include_object(object, name, type_, reflected, compare_to):
+        if type_ == 'table':
+            return object.schema == _schema
+        else:
+            return object.table.schema == _schema
+    if _schema:
+        settings.update({
+            'target_metadata': Base.metadata,
+            'include_schemas': True,
+            'include_object': include_object
+        })
```

Comment:
For what is this code for?

Reply:
As the comment says, this is the autogenerate config, for details see:
https://alembic.sqlalchemy.org/en/latest/autogenerate.html
https://alembic.sqlalchemy.org/en/latest/api/runtime.html#alembic.runtime.environment.EnvironmentContext.configure.params.target_metadata

Reply:
OK, thanks :-)